docs/06-manual-builds.md

```diff
@@ -0,0 +1,161 @@
+
+## Manually building for iOS
+
+### Prerequisites
+* First setup XCode and [CocoaPods](#setup-of-cocoapods).
+* In order to set up certificates you also need [to set up Fastlane](08-cicd.md#setup-of-fastlane).
+* Read about [Apple Certificates](10-apple-certifcates.md)
+* Setup CocoaPods
+
+  CocoaPods is used for dependency management of the native libraries.
+  First you have to make sure that Ruby is working correctly. We advise to use RVM as mentioned in the [setup of Fastlane](08-cicd.md#setup-of-fastlane). After running this setup you should also be able to do `bundle exec pod --version`.
+    
+  *Note: `bundle exec pod install` uses the versions from the `Podfile.lock`. `bundle exec pod update` updates the `Podfile.lock`.*
+
+
+### Setup Certificates
+
+*Note: This step is required in order to build the project for iOS*
+
+First you have to prepare a [few environment variables](08-cicd.md#environment-variables-and-dependencies):
+
+```bash
+export FASTLANE_USER=<secret>
+export FASTLANE_PASSWORD=<secret>
+export MATCH_PASSWORD=<secret>
+```
+
+Finally, install the certificates locally:
+
+```bash
+cd ios/
+bundle exec fastlane certificates
+```
+
+*Note: After setting up the certificates you can also start to use XCode to build the app.*
+
+### Run the app in a Simulator
+
+```bash
+yarn
+cd ios/
+bundle exec pod install
+cd ..
+yarn start
+yarn ios
+```
+
+### Run the app on a Device or Simulator via XCode
+
+```bash
+yarn
+cd ios/
+bundle exec pod install
+cd ..
+yarn start
+```
+
+Now start XCode and open `ios/Integreat.xcworkspace`. Then you can build and start the app on your device or simulator.
+
+### Building the App with a Distribution Certificate
+Build the app:
+
+```bash
+yarn
+cd ios/
+bundle exec pod install
+bundle exec fastlane build
+```
+
+Fastlane should report where the build artifacts are. These can be uploaded to App Store Connect or distributed via another way.
+
+## Manually building for Android
+
+If you do not have Fastlane installed [you can skip to the next section](#quick-builds-using-a-test-signing-key), which does not require FL.
+
+### Prerequisites
+* Setup of the Android SDK
+* In order to set up production keystores you also need [to set up Fastlane](08-cicd.md#setup-of-fastlane).
+
+### Building the App for Play Store
+
+Firstly you have to prepare a [few environment variables](08-cicd.md#environment-variables-and-dependencies):
+
+```bash
+export CREDENTIALS_GIT_REPOSITORY_URL=<secret>
+export CREDENTIALS_DIRECTORY_PATH=/tmp/credentials
+export CREDENTIALS_KEYSTORE_PASSWORD=<secret>
+export CREDENTIALS_KEYSTORE_PATH=/tmp/credentials/<secret>.enc
+export KEYSTORE_PATH=/tmp/keystore.jks
+```
+
+Setup your JKS now:
+
+```bash
+cd android/
+bundle exec fastlane keystore
+```
+
+Finally, provide some more information about unlocking the JKS:
+
+```bash
+export KEYSTORE_KEY_ALIAS=<secret>
+export KEYSTORE_PASSWORD=<secret>
+export KEYSTORE_KEY_PASSWORD=<secret>
```

Comment:
I'd move this to a separate sub like in the ios part. I'd also apply the suggestions made for the ios certificate setup part.

Reply:
I restructured the section